src/components/new-request/search.vue

```diff
@@ -135,69 +135,32 @@
           </template>
         </v-select>
       </v-col>
-      <v-col :cols="getIsXproMras ? 8 : 12" :class="{ 'pl-3': (getIsXproMras && !isFederal) }">
+      <v-col :class="{ 'pl-3': (getIsXproMras && !isFederal), 'pr-3': !getIsXproMras && showDesignationSelect}"
+             :cols="showDesignationSelect || (getIsXproMras) ? '8' : '12'">
         <NameInput v-if="!isFederal"
                    :class="inputCompClass"
                    :is-mras-search="(getIsXproMras && !noCorpNum)"
                    id="name-input-component"
                    class="pa-0"
                    @emit-corp-num-validity="corpNumValid = $event"/>
-        <p v-else class="pl-3 mt-n2 text-body-2">Federally incorporated businesses do not need a Name Request. You may
+        <p v-else class="pl-3 text-body-2">Federally incorporated businesses do not need a Name Request. You may
           register  your extraprovincial business immediately using its existing name at Corporate Online.</p>
```

Comment:
Ask the UX folks if they want a link to Corp Online here.

Reply:
ha - too late there's no time, could be an enhancement added in the UI ticket for updating this component

Reply:
ok